velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -270,11 +278,24 @@ TEST_F(HdfsFileSystemTest, missingPort) {
 
 TEST_F(HdfsFileSystemTest, missingFileViaReadFile) {
   try {
-    struct hdfsBuilder* builder = hdfsNewBuilder();
-    hdfsBuilderSetNameNode(builder, localhost.c_str());
-    hdfsBuilderSetNameNodePort(builder, std::stoi(hdfsPort));
-    auto hdfs = hdfsBuilderConnect(builder);
-    HdfsReadFile readFile(hdfs, "/path/that/does/not/exist");
+    filesystems::arrow::io::internal::LibHdfsShim* driver;
+    auto status = filesystems::arrow::io::internal::ConnectLibHdfs(&driver);
```

Comment:
These 10 lines of code seem to be repeated for each test. Should we use a util function?

Reply:
@majetideepak Yes. Add new function `connectHdfsDriver`.